Jenkinsfile

```diff
@@ -306,7 +306,9 @@ for (config in configs) {
             builders[combinedName] = {
                 node(label) {
                     stage(combinedName) {
-                        docker.image(imageName).inside {
+                        buildImage = docker.image(imageName)
```

Comment:
Doh. Please put a `def` on this assignment. And then do a PR to all of the others fixing this :-(